src/EnergyPlus/DaylightingManager.cc

```diff
@@ -7422,7 +7408,6 @@ void DayltgElecLightingControl(EnergyPlusData &state)
         } else { // controls not currently available
             TotReduction = 1.0;
         }
-        thisDaylightControl.PowerReductionFactor = TotReduction;
```

Comment:
This deletion confused me at first, but then I realized it's a reference `Real64 &TotReduction = thisDaylightControl.PowerReductionFactor;`